pyproject.toml

```
@@ -29,10 +28,21 @@
         name = "Improved Documentation"
         showcontent = true
 
+    [[tool.towncrier.type]]
+        directory = "removal"
+        name = "Removals"
+        showcontent = true
+
+    [[tool.towncrier.type]]
+        directory = "deprecation"
+        name = "Deprecations"
+        showcontent = true
+
     [[tool.towncrier.type]]
         directory = "misc"
         name = "Misc"
-        showcontent = true
+        showcontent = false
```

```suggestion
        showcontent = true
```
I think I'd rather keep this `true`.